sandpack-react/src/presets/Sandpack.tsx

```diff
@@ -186,6 +188,21 @@ export const Sandpack: SandpackInternal = ({
     ? { className: THEME_PREFIX + "-preset-column", style: rightColumnStyle }
     : {};
 
+  const CodeEditor = () => {
```

Comment:
I think this will make this component re-render all the time, specially when there a state change, like the resize handler